build/toolchain/linux/BUILD.gn

```diff
@@ -21,16 +22,20 @@ if (use_goma) {
   compiler_prefix = ""
 }
 
-if (host_cpu == "arm64") {
-  rebased_clang_dir =
-      rebase_path("//buildtools/linux-arm64/clang/bin", root_build_dir)
+if (is_host_glibc) {
+  if (host_cpu == "arm64") {
+    rebased_clang_dir =
+        rebase_path("//buildtools/linux-arm64/clang/bin", root_build_dir)
+  } else {
+    rebased_clang_dir =
+        rebase_path("//buildtools/linux-x64/clang/bin", root_build_dir)
+  }
 } else {
-  rebased_clang_dir =
-      rebase_path("//buildtools/linux-x64/clang/bin", root_build_dir)
+  rebased_clang_dir = ""
```

Comment:
Is this a fair assumption for uclibc?

Honestly I'm not sure if we've got uclibc users any more; we definitely did have internally at one point, but my recollection is that it was an experimental usecase that no longer exists. @zanderso will probably know.